src/backend/protocols/go.imap/worker.go

```diff
@@ -83,12 +84,17 @@ func NewWorker(config WorkerConfig, id uint8) (worker *Worker, err error) {
 		}
 	}
 
+	// init Sender
 	return &w, nil
 }
 
 func (worker *Worker) Start() error {
 	var err error
-	(*worker).NatsSub, err = worker.NatsConn.QueueSubscribe(worker.Config.NatsTopic, worker.Config.NatsQueue, worker.natsMsgHandler)
+	(*worker).NatsSubs[0], err = worker.NatsConn.QueueSubscribe(worker.Config.NatsTopicFetcher, worker.Config.NatsQueue, worker.natsMsgHandler)
+	if err != nil {
+		return err
+	}
+	(*worker).NatsSubs[1], err = worker.NatsConn.QueueSubscribe(worker.Config.NatsTopicSender, worker.Config.NatsQueue, worker.natsMsgHandler)
```

Comment:
why subscribe twice on the same queue ?

Reply:
that's actualy two different subjects : Fetcher and Sender

Reply:
Ok my bad